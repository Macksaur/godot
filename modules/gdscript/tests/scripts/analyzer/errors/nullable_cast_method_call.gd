func test():
    var nullable_string: String? = null
    (nullable_string as String).begins_with("") # Should not error
    (nullable_string as String?).begins_with("") # Should error
