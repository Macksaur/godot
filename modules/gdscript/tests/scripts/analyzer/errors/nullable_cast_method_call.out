GDTEST_ANALYZER_ERROR
Potentially unsafe for the nullable cast expression "nullable_string as String?" to call "begins_with()".
